### DISCUSSION TITLE
What's testscenario used for?

### DISCUSSION THREAD

**Discussion by KaiserKarel** [quasar-finance/quasar#261] [General]
https://github.com/quasar-finance/quasar/blob/cc7d85adc2a76152cee28e586d1c1fe00585c353/smart-contracts/packages/quasar-bindings/src/msg.rs#L9

**Comment by DevKob** [comment]
"`TestScenario`" is a technical concept used to define and manage different testing situations for the software being developed. It's a way for developers to comprehensively verify that the software works as expected across a range of scenarios before it's deployed.
